### PR TITLE
Refactor/deployment

### PR DESCRIPTION
**Pull request by Jayashsatolia403**
### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/blobber/zcn.go [hunk 1]
```diff
@@ -31,8 +31,8 @@ func registerOnChain() error {
 		return err
 	}
 
-	// setup blobber (add or update) on the blockchain (multiple attempts)
-	for i := 1; i <= 10; i++ {
+	// setup blobber (add) on the blockchain (multiple attempts)
+	for i := 1; i <= 120; i++ {
```
Comment (dabasov) [inline code/go/0chain.net/blobber/zcn.go]: can we have this loop to be unlimmitted?

File: code/go/0chain.net/blobbercore/handler/handler.go [hunk 2]
```diff
@@ -135,104 +135,116 @@ func SetupSwagger() {
...
+
 /*setupHandlers sets up the necessary API end points */
 func setupHandlers(r *mux.Router) {
 	ConfigRateLimits()
 	r.Use(UseRecovery, UseCors)
 
 	//object operations
 	r.HandleFunc("/v1/connection/create/{allocation}",
-		RateLimitByGeneralRL(common.ToJSONResponse(WithConnection(CreateConnectionHandler)))).
+		RateLimitByGeneralRL(WithBlobberRegisteredCondition(common.ToJSONResponse(WithConnection(CreateConnectionHandler))))).
```
Comment (dabasov) [inline code/go/0chain.net/blobbercore/handler/handler.go]: hide into middleware

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  |   |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  |   |
| zwalletcli  |   |
| 0chain  |   |
| blobber  | refactor/deployment  |
| authorizer  |   |
| 0box  |   |
| 0dns  |   |

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**